boot/bootutil/src/bootutil_priv.h

```diff
@@ -268,6 +268,9 @@ struct boot_loader_state {
 fih_ret bootutil_verify_sig(uint8_t *hash, uint32_t hlen, uint8_t *sig,
                             size_t slen, uint8_t key_id);
 
+fih_ret bootutil_verify_img(const uint8_t *img, uint32_t size,
+                            uint8_t *sig, size_t slen, uint8_t key_id);
+
```

Comment:
nit: I would appreciate code comment on difference to above function.

Reply:
Added comment.